### PR TITLE
support colors on windows

### PR DESCRIPTION
**Pull request by szabolcsdombi**
Solving #6 and #1 

- support colors on windows

image

### CODE REVIEW

File: tbvaccine/__main__.py [hunk 1]
```diff
@@ -19,8 +19,49 @@
...
+        set_console_mode_ok = SetConsoleMode(stdout_handle, console_mode)
+        if not set_console_mode_ok:
+            return False
+
+    # Successful initialization
+    return True
+
 with open(script_path) as script_file:
+    if not initialize():
+        pass # TODO: better not add colors then
```
Comment (szabolcsdombi) [inline tbvaccine/__main__.py]: Without colors further calls are pointless.
Exception or continue without colors?

Comment (skorokithakis) [reply tbvaccine/__main__.py]: Continue without colors if colors are not supported, yes. We still want the traceback to show.

**Comment by skorokithakis** [comment]
Thank you! 0.1.2 released.

**Comment by szabolcsdombi** [comment]
Thank you!

**Pull request merged**